### PR TITLE
Create a better solution for define multi line fields

### PR DESCRIPTION
**Pull request by Freeman6475**
Fixes https://github.com/JabRef/jabref/issues/6448

grafik

- [ ] Change in `CHANGELOG.md` described in a way that is understandable for the average user (if applicable)
- [ ] Tests created for changes (if applicable)
- [x] Manually tested changed features in running JabRef (always required)
- [x] Screenshots added in PR description (for UI changes)
- [ ] [Checked developer's documentation](https://devdocs.jabref.org/): Is the information available and up to date? If not, I outlined it in this pull request.
- [ ] [Checked documentation](https://docs.jabref.org/): Is the information available and up to date? If not, I created an issue at <https://github.com/JabRef/user-documentation/issues> or, even better, I submitted a pull request to the documentation repository.

### CODE REVIEW

**Comment by ThiloteE** [comment]
To ease organizational workflows I have linked the pull-request to the issue with syntax as described in https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue

> <h2 id="linking-a-pull-request-to-an-issue-using-a-keyword">Linking a pull request to an issue using a keyword</h2>
> <p>You can link a pull request to an issue by using a supported keyword 
> in the pull request's description or in a commit message. The pull 
> request <strong>must be</strong> on the default branch.</p>
> <ul>
> <li>close</li>
> <li>closes</li>
> <li>closed</li>
> <li>fix</li>
> <li>fixes</li>
> <li>fixed</li>
> <li>resolve</li>
> <li>resolves</li>
> <li>resolved</li>
> </ul>
> <p>If you use a keyword to reference a pull request comment in another 
> pull request, the pull requests will be linked. Merging the referencing 
> pull request also closes the referenced pull request.</p>
> <p>The syntax for closing keywords depends on whether the issue is in the same repository as the pull request.</p>

**Comment by Siedlerchr** [comment]
As we talked about this, I summarize this here again for the others:

Next steps:
create a checkbox column for the multiline check

see for an example how it is implemented
https://github.com/JabRef/jabref/blob/c02d75b9fb8025de5dc461033fb4276c964dd519/src/main/java/org/jabref/gui/slr/ManageStudyDefinitionView.java#L219

**Comment by Siedlerchr** [comment]
TODO: How to adjust the serializer/parser to recognize the field properties. Store it in prefs as is?

**Comment by Siedlerchr** [comment]
NON_WRAPPABLE_FIELDS: 
FieldContentFormatterPreferences extract  to BibEntryPreferences 
Custom Entry types: move to preferences 

Call with Carl: Still use the existing hack

**Comment by Siedlerchr** [comment]
I will create a follow up PR for moving the CustomEntryType to the preference tab

File: src/main/java/org/jabref/gui/customentrytypes/CustomizeEntryTypeDialog.fxml [hunk 1]
```diff
@@ -55,19 +55,21 @@
...
                                 <TableColumn fx:id="fieldNameColumn"
-                                             minWidth="150.0" prefWidth="-1.0" text="%Field"/>
+                                             minWidth="120.0" prefWidth="-1.0" text="%Field"/>
                                 <TableColumn fx:id="fieldTypeColumn"
-                                             minWidth="100.0" prefWidth="100.0" text="%Field type"/>
+                                             minWidth="130.0" prefWidth="150.0" text="%Field type"/>
                                 <TableColumn fx:id="fieldTypeActionColumn"
                                              maxWidth="40.0" minWidth="40.0" resizable="false"/>
+                                <TableColumn fx:id="fieldTypeMultilineColumn" text="%Multiline"
+                                             prefWidth="60.0" minWidth="50.0" resizable="true"/>
```
Comment (calixtus) [inline src/main/java/org/jabref/gui/customentrytypes/CustomizeEntryTypeDialog.fxml:72]: Not so happy with the arrangement of the checkboxes and buttons.
The remove button should be the most right aligned button. the checkbox-column is too wide. Maybe rotate the label of the TableColumn 90 degrees to display it vertically? (https://stackoverflow.com/questions/27666087/vertical-column-text-in-tableview)

Comment (Siedlerchr) [reply src/main/java/org/jabref/gui/customentrytypes/CustomizeEntryTypeDialog.fxml:72]: I found a different solution for vertical alignment https://stackoverflow.com/a/74863464
In general this looks good, however, there seems to be some padding or margin missing.. Unfortunately I could not get scenic view to run on mac :(
grafik

Comment (calixtus) [reply src/main/java/org/jabref/gui/customentrytypes/CustomizeEntryTypeDialog.fxml:72]: I will take a look

Comment (calixtus) [reply src/main/java/org/jabref/gui/customentrytypes/CustomizeEntryTypeDialog.fxml:72]: can you commit your changes?

Comment (Siedlerchr) [reply src/main/java/org/jabref/gui/customentrytypes/CustomizeEntryTypeDialog.fxml]: Pushed to the wrong repo^^

Comment (Siedlerchr) [reply src/main/java/org/jabref/gui/customentrytypes/CustomizeEntryTypeDialog.fxml]: Fixed it  now 
grafik

Comment (calixtus) [reply src/main/java/org/jabref/gui/customentrytypes/CustomizeEntryTypeDialog.fxml]: I'll look into it tomorrow. There are some UI tweaks I'm thinking of. 🤔

**Comment by calixtus** [comment]
grafik

**Review by Siedlerchr — APPROVED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -11,12 +11,15 @@ Note that this project **does not** adhere to [Semantic Versioning](http://semve
 
 ### Added
 
+
```
Comment (calixtus) [inline CHANGELOG.md:14]: Duplicate changelog entry

**Pull request merged**